src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -251,5 +233,76 @@ canvas {
 }
 
 .asset-error {
+    font-size: 85%;
+    font-weight: 500;
     margin: auto;
 }
+
+.badge-visited, .badge-tagged {
+    color: #fff;
+    position: absolute;
+    z-index: 2;
+    right: 0.5em;
+    top: 0.5em;
+    text-shadow: 0px 0px 4px #000
```

Comment:
1px or 2px shadow is more in keeping with the rest of the icons - just enough to hold the edges.

Reply:
This is more of a glow radius vs an actual shadow to allow the white icon to stand out better on top of the lighter (orange) badge color.